src/Grid.js

```diff
@@ -0,0 +1,133 @@
+import Cell from './Cell.js';
+
+export default class Grid
+{
+	constructor()
+	{
+		this.cycle = 0;
+		this.cells = [];
+		this.cyclePositions = [];
+	}
+
+	/**
+	 * @param {number} xPosition
+	 * @param {number} yPosition
+	 * @return {string}
+	 */
+	getCellKey(xPosition, yPosition)
+	{
+		return `${xPosition},${yPosition}`;
+	}
+
+	/**
+	 * @param {number} xPosition
+	 * @param {number} yPosition
+	 * @param {Cell} cell
+	 * @param {Grid}
+	 */
+	setCell(xPosition, yPosition, cell)
+	{
+		let cellKey = this.getCellKey(xPosition, yPosition);
+		this.cells[cellKey] = cell;
+
+		if (!cell.isAlive()) {
+			return this;
+		}
+
+		for (let xIndex = xPosition - 1; xIndex <= (xPosition + 1); xIndex++) {
+			for (let yIndex = yPosition - 1; yIndex <= (yPosition + 1); yIndex++) {
+				cellKey = this.getCellKey(xIndex, yIndex);
+				if (typeof this.cells[cellKey] === 'undefined') {
+					this.cells[cellKey] = new Cell(false);
```

Comment:
@dominiktilp basically i hate the idea to recompute a board 1000x1000 cell by cell, when there are just a few living cells. So instead I only store the cells that can be affected in next cycle (living + neightbours).
It is truth that currently my board has no dimensions, so the size is only limited by positions of actually living cells. This can obviously lead to issues when we get generating patterns going out of integer bounds - I plan to limit the gird in the future.